### ISSUE TITLE
How to fix error code 17 after every update

### ISSUE THREAD

**Issue by zakblacki** [SpotX-Official/SpotX#599]
### 🧐 Have you read the FAQ and viewed similar issues?

- [X] I have read the FAQ
- [X] I have checked older issues, open and closed
- [X] I did a clean install of SpotX as <a href="https://telegra.ph/SpotX-FAQ-09-19#1.-Clean-installation">described in the FAQ</a>

### Сountry of your account

Algeria

### What is your Spotify Plan?

Free

### Do you have Spicetify installed?

No

### ℹ Computer information

```markdown
- Spotify: 1.2.37.701
- Windows: Windows 10 Pro 22H2 OS 19045.4412
- PowerShell: 5.1.19041.4412
```

### 📝 Description

Everytime I update or do clean installation I get the following error Error code : 17
all i remember this issue appeared after I started using SpotX 
the weird thing is that regardless of the error my spotify version still get updated.

### 📸 Screenshots

![image](https://github.com/SpotX-Official/SpotX/assets/29728429/387eb06d-4a54-4d9f-834e-bc2041098491)

**Comment by amd64fox** [comment]
I can't reproduce your error on my pc, show your pc specs, example:

- CPU - Intel Core i7-4790K 
- GPU - Integrated Graphics
- RAM - DDR3 16 Gb
- Spotify is located on - SSD

**Comment by zakblacki** [comment]
> I can't reproduce your error on my pc, show your pc specs, example:
> 
> * CPU - Intel Core i7-4790K
> * GPU - Integrated Graphics
> * RAM - DDR3 16 Gb
> * Spotify is located on - SSD

CPU - i7- 7700k 
GPU - AMDRX 480 8GB 
RAM - DDR4 16 Gb
Spotify on SSD C:  %AppData%/Roaming/Spotify/

Please note this has been happening for awhile in previous version as well it seem to be a common issue 
https://community.spotify.com/t5/Desktop-Windows/error-code-17/td-p/25431

**Comment by amd64fox** [comment]
> Please note this has been happening for awhile in previous version as well it seem to be a common issue https://community.spotify.com/t5/Desktop-Windows/error-code-17/td-p/25431

Yeah, I already googled this problem, it doesn’t give an answer to the solution to the current problem

The essence of this error is that after running the full Spotify installer and completing its process, it tries to launch the installed Spotify. SpotX attempts to intercept this launch and forcefully terminate it in order to immediately proceed with file modification.

However, it seems that in this part of the logic, the SpotX code is not executing correctly for you.

Perhaps I should find another way to install the client. I saw that `skip-app-launch` flag appeared in their installers not long ago, I think I should start using it soon

**Comment by princessmisery** [comment]
Hello I'm so sorry to drop into this convo, but I can't find where I can make a new post to ask a question. I tried the discussion area too. I am so super frustrated because even without this patcher, I keep getting this error that Spotify can't play the song right now and I can play it if I have the file on my computer. I'm sure someone around here has ran into this. I cannot for the life of me figure out how to fix this. I've tried so many random weird things that people say work. Please please can someone help me with that? I'd really like to listen to my liked songs SO BAD. Thank you so much and so sorry to just randomly post here.

**Comment by amd64fox** [comment]
@princessmisery 
you could have found this [link](https://community.spotify.com/t5/FAQs/quot-Can-t-play-the-current-track-quot-error/ta-p/1279790/redirect_from_archived_page/true#M86679) in the faq

**Comment by TheNoobiCat** [comment]
Cannot reproduce on:
Ryzen 5 5600g
Rtx 2060
DDR4 48GB RAM
Spotify on nvme
